requirements.txt

```diff
@@ -1,4 +1,10 @@
-
+# Django/Framework Packages
+django>=1.4.12,<=1.4.22
+django-model-utils==2.3.1
+South>=0.7.6
+djangorestframework>=2.3.5,<=2.3.14
+pytz>=2012h
+pycrypto>=2.6
```

Comment:
@chrisndodge I saw that you recently moved these requirements to a separate file to resolve conflicts with edx-platform.  What errors were you running into?  When I move these back and bump the `setup.py` version number, everything seems to install correctly.